components/org.wso2.carbon.micro.integrator.extensions/org.wso2.carbon.micro.integrator.management.apis/src/main/java/org/wso2/carbon/micro/integrator/management/apis/LoggingResource.java

```diff
@@ -0,0 +1,163 @@
+package org.wso2.carbon.micro.integrator.management.apis;
+
+import org.apache.log4j.Level;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import org.apache.synapse.MessageContext;
+import org.apache.synapse.commons.json.JsonUtil;
+import org.apache.synapse.core.axis2.Axis2MessageContext;
+import org.json.JSONObject;
+
+import java.util.Enumeration;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+public class LoggingResource extends ApiResource {
+
+    private static String[] logLevels = new String[]{"OFF", "TRACE", "DEBUG", "INFO", "WARN", "ERROR", "FATAL"};
```

Comment:
```suggestion
    private static final String[] logLevels = new String[]{"OFF", "TRACE", "DEBUG", "INFO", "WARN", "ERROR", "FATAL"};
```

Reply:
Shall we use an array of `org.apache.log4j.Level` objects instead of Strings?

Reply:
Fixed in 5f84b6a